app/repository/pg/conndb.go

```diff
@@ -0,0 +1,36 @@
+package pg
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/go-devs-ua/octagon/cfg"
+)
+
+type DB struct {
```

Comment:
Heh, you don't need these structs anymore.

Reply:
Oh... forgot about it :D